crates/world-id-core/src/types.rs

```diff
@@ -72,111 +73,151 @@ pub struct RpRequest {
 
 /// The request to create a new World ID account.
 #[cfg(feature = "authenticator")]
-#[derive(Debug, Serialize, Deserialize)]
+#[derive(Debug, Serialize, Deserialize, ToSchema)]
 pub struct CreateAccountRequest {
     /// The recovery address.
+    #[schema(value_type = String, format = "hex")]
     pub recovery_address: Option<Address>,
     /// The addresses of the authenticators.
+    #[schema(value_type = Vec<String>, format = "hex")]
     pub authenticator_addresses: Vec<Address>,
     /// The compressed public keys of the authenticators.
+    #[schema(value_type = Vec<String>, format = "hex")]
     pub authenticator_pubkeys: Vec<U256>,
     /// The offchain signer commitment.
+    #[schema(value_type = String, format = "hex")]
     pub offchain_signer_commitment: U256,
 }
 
 /// The request to update an authenticator.
 #[cfg(feature = "authenticator")]
-#[derive(Debug, Serialize, Deserialize)]
+#[derive(Debug, Serialize, Deserialize, ToSchema)]
 pub struct UpdateAuthenticatorRequest {
     /// The account index.
+    #[schema(value_type = String, format = "hex")]
```

Comment:
double check this, I believe these may be getting serialized as decimals. I have a preference for hex but not sure we're serializing as hex right now